tests/admin_utils.hpp

```diff
@@ -31,6 +30,8 @@
 #include <sstream>
 #include <vector>
 
+#include <sys/wait.h>
```

Comment:
is this used anywhere?

Reply:
Yes, this is where `waitpid` comes from.